api/src/test/java/org/pmiops/workbench/api/BigQueryBaseTest.java

```diff
@@ -0,0 +1,107 @@
+package org.pmiops.workbench.api;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.cloud.bigquery.*;
```

Comment:
Nit: don't import *. Makes it hard to tell what package a class is coming from.

Reply:
I just changed my intellij setting to keep it from using the wildcard.